drivers/net/bnxt/bnxt_hwrm.c

```diff
@@ -2014,8 +2020,8 @@ int bnxt_hwrm_allocate_vfs(struct bnxt *bp, int num_vfs)
 		rc = -ENOMEM;
 		goto error_free;
 	}
-	for (ui = 0; ui < req_buf_sz; ui += rte_eal_get_physmem_size()) {
-		rte_mem_lock_page(((char *)bp->pf.vf_req_buf) + ui);
+	for (sz = 0; sz < req_buf_sz; sz += getpagesize()) {
```

Comment:
Yes, this is a separate bug fix.  I honestly don't know how I got this bit so wrong.